scripts/tests/test_aggregate_tablero_replay_breakdown.py

```
@@ -0,0 +1,175 @@
+            "--output-tsv",
+            str(out_tsv),
+        ]
+    )
+    return json.loads(out_json.read_text(encoding="utf-8"))
+
+
+class AggregateTableroReplayBreakdownTests(unittest.TestCase):
+    def write_runs(self, tmp: Path, totals_per_run: list[dict[str, float]]) -> list[Path]:
+        paths: list[Path] = []
+        for index, family_totals in enumerate(totals_per_run, start=1):
+            wrapped = {family: total for family, total in family_totals.items()}
```

Ruff (0.15.11)

[warning] Unnecessary dict comprehension (rewrite using `dict()`)

Rewrite using `dict()`

(C416)